bundles/org.openhab.binding.bticinosmarther/README.md

```diff
@@ -0,0 +1,328 @@
+* Should you have specified a wrong "Reply URL" attribute in your application, go to the Legrand portal and correct it accordingly then resubmit the application for approval.
+
+## Supported Things
+
+All BTicino Smarther Chronothermostat device models should be discoverable through this binding:
+
+* Flush mounting installation item (X8000)
+* Wall installation item (X8000W)
+
+If you can control them from BTicino Thermostat mobile app on your iPhone/Android you should be able to add it as a thing.
+
+## Discovery
+
+As long as BTicino Smarther Chronothermostat devices are available in the locations registered on the user account configured with the bridge they should show up whenever you initiate discovery of things.
+
+If no devices are showing up, try to connect to the device(s) from your smartphone to make sure the device(s) are in use by your user account.
+
+The discovery of devices in the Smarther API is based on what is known by Legrand.
+There is difference between e.g. smartphones and computers which can discover devices on the local network and the Smarther API which is not able to do so; it only knows about a device if your account is currently associated with the device.
+
+## Channels
+
+### Bridge
+
+The channels on the bridge are the ones used to get details of current communication with Smarther API on the Legrand account associated with the bridge.
+
+**Status Channels:**
+
+The following channels represent the current operational status of the bridge and must all be referenced with the `status#` prefix.
+
+| Channel Type ID   | Item Type            | Read/Write | Description                                                                            | Type     |
+|-------------------|----------------------|------------|----------------------------------------------------------------------------------------|----------|
+| apiCallsHandled   | Number               | Read-only  | The total number of API calls handled by the bridge                                    | Common   |
+| notifsReceived    | Number               | Read-only  | The total number of push notifications received by the bridge                          | Common   |
+| notifsRejected    | Number               | Read-only  | The total number of push notifications rejected by the bridge                          | Common   |
+
+**Configuration Channels:**
+
+The following channels represent convenience configuration channels for the bridge and must all be referenced with the `config#` prefix.
+
+| Channel Type ID   | Item Type            | Read/Write | Description                                                                            | Type     |
+|-------------------|----------------------|------------|----------------------------------------------------------------------------------------|----------|
+| fetchConfig       | Switch               | Read-write | Trigger to manually fetch updated bridge configuration from Smarther API               | Advanced |
```

Comment:
This is not clear to me. What is the "bridge configuration" that is mentioned here? The bridge configuration should be set by the user, not by the external device.

Reply:
Maybe "bridge" configuration is a bit misleading. The bridge automatically gathers the remote device network topology from API. This is done on a periodical basis, as the API itself has a daily usage limit. The trigger is used to fetch the same information on demand. I can eventually change the description to "Trigger to manually fetch the updated remote device network topology"...

Reply:
If manually triggering the fetch is a use case, I it ok for me to have such a channel (re-initializing the bridge would rather be a workaround for the user). So yes, if you could adapt the description to use vocabulary that does not collide with openHAB stuff, that would be nice!